recipes/protobuf/all/conanfile.py

```diff
@@ -176,7 +176,7 @@ def _patch_sources(self):
             tools.replace_in_file(
                 protobuf_config_cmake,
                 "add_custom_command(",
-                ("set(CUSTOM_DYLD_LIBRARY_PATH ${CONAN_LIB_DIRS} ${Protobuf_LIB_DIRS} ${Protobuf_LIB_DIRS_RELEASE} ${Protobuf_LIB_DIRS_DEBUG} ${Protobuf_LIB_DIRS_RELWITHDEBINFO} ${Protobuf_LIB_DIRS_MINSIZEREL})\n"
+                ("set(CUSTOM_DYLD_LIBRARY_PATH ${CONAN_LIB_DIRS} ${protobuf_LIB_DIRS}\n"
```

Comment:
it will break cmake_find_package & cmake_find_package_multi

Reply:
> `protobuf_LIB_DIRS` doesn't exist in cmake_find_package_multi generator, so don't remove variables with build type suffix please.
OK. I'll revert this change later. And I will do some more tests.
 
> Are you sure it's `protobuf_LIB_DIRS` instead of `Protobuf_LIB_DIRS` in file generated by `cmake_find_package`?
Yes, I have tested it on my OSX.

My CMakeLists.txt:

```cmake
conan_cmake_configure(REQUIRES protobuf/xxx
                      GENERATORS cmake_find_package)
# ...
find_package(Protobuf REQUIRED)
```

CMake command generated `FindProtobuf.cmake`, and the variable in it was `protobuf_LIB_DIRS`.
PS: cmake generator is Ninja.